autotst/calculator/statmech.py

```diff
@@ -375,32 +262,13 @@ def write_ts_input(self, transitionstate):
         transitionstate.update_coords_from("ase")
 
         output = ['#!/usr/bin/env python',
-                  '# -*- coding: utf-8 -*-', '', 'atoms = {']
+                  '# -*- coding: utf-8 -*-']
 
-        atom_dict = self.get_atoms(conformer=transitionstate)  # need to fix
-
-        for atom, count in atom_dict.items():
-            output.append("    '{0}': {1},".format(atom, count))
-        output = output + ['}', '']
-
-        bond_dict = self.get_bonds(conformer=transitionstate)  # need to fix
-        if bond_dict != {}:
-            output.append('bonds = {')
-            for bond_type, num in bond_dict.items():
-                output.append("    '{0}': {1},".format(bond_type, num))
-
-            output.append("}")
-        else:
-            output.append('bonds = {}')
         transitionstate.rmg_molecule.update_multiplicity()
 
-        external_symmetry = transitionstate.calculate_symmetry_number()
-
         output += ["",
                    "linear = False",
```

Comment:
I think we should remove `linear` and `opticalIsomers` here too

Reply:
Removed those sections, let's see how the tests go.